systems/plants/TimeSteppingRigidBodyManipulator.m

```diff
@@ -252,6 +248,10 @@ function checkDirty(obj)
     end
 
     function [obj,z,Mqdn,wqdn,dz,dMqdn,dwqdn] = solveLCP(obj,t,x,u)
+      if (nargout<5 && obj.gurobi_present && obj.manip.only_loops && obj.manip.mex_model_ptr~=0 && ~obj.position_control)
+        [obj,z,Mqdn,wqdn] = solveMexLCP(obj,t,x,u);
+      end
```

Comment:
Wait.  Isn't this still solving LCP twice?  I think we need a return statement here.